net_ip_socket.go

```diff
@@ -24,6 +24,12 @@ import (
 	"strings"
 )
 
+var (
+	// isUDP is used to determine the type of network socket UDP or TCP.
+	// Further its value will be used to parse UDP-specific fields from /proc/net/udp{,6} files.
+	isUDP bool
```

Comment:
Why is this a global variable?

Reply:
You are right, I will change to a local variable.